### PR TITLE
Fix the Passport scrollbar

### PR DESCRIPTION
**Pull request by jonbell-lot23**
We have an issue where the scrollbar is tucking behind the animated gif. This should fix it.
https://linear.app/replay/issue/DES-647/scrollbar-still-broken-in-passport

(Note: there are other flubs to deal with, like how the documentation icon is overflowing on other elements. Will be dealing with those as well, stay tuned)

### CODE REVIEW

File: src/ui/components/Passport/Passport.tsx [hunk 1]
```diff
@@ -283,50 +283,29 @@ const Passport = () => {
...
+            ))}
           </div>
         </div>
       </div>
+
+      <div className="p-1">
+        <img
+          src={selectedItem.videoUrl}
+          className={`${styles.videoExample} w-full object-cover`}
+          ref={videoExampleRef}
```
Comment (bvaughn) [inline src/ui/components/Passport/Passport.tsx:306]: I found myself curious why you were using a ref here.

Looks like there are duplicate effects:
https://github.com/replayio/devtools/blob/c134c0cdbb8359fdd2526ae1ca10472f6b7bdc72/src/ui/components/Passport/Passport.tsx#L47-L68

You could consolidate those:
```js
useLayoutEffect(() => {
  const videoExample = videoExampleRef.current;
  if (videoExample) {
    const updateHeight = () => {
      setVideoHeight(videoExample.offsetHeight);
    };

    window.addEventListener("resize", updateHeight);

    return () => {
      window.removeEventListener("resize", updateHeight);
    };
  }
}, []);
```

So...
```diff
diff --git a/src/ui/components/Passport/Passport.tsx b/src/ui/components/Passport/Passport.tsx
index 59c8e3130..5e17b6387 100644
--- a/src/ui/components/Passport/Passport.tsx
+++ b/src/ui/components/Passport/Passport.tsx
@@ -44,27 +44,19 @@ const Passport = () => {
   const videoExampleRef = useRef<HTMLImageElement>(null);
   const [videoHeight, setVideoHeight] = useState<number | null>(null);
 
-  useEffect(() => {
-    if (videoExampleRef.current) {
-      const videoHeight = videoExampleRef.current.offsetHeight;
-      setVideoHeight(videoHeight);
-    }
-  }, [videoExampleRef]);
+  useLayoutEffect(() => {
+    const videoExample = videoExampleRef.current;
+    if (videoExample) {
+      const updateHeight = () => {
+        setVideoHeight(videoExample.offsetHeight);
+      };
 
-  useEffect(() => {
-    const updateHeight = () => {
-      if (videoExampleRef.current) {
-        const videoHeight = videoExampleRef.current.offsetHeight;
-        setVideoHeight(videoHeight);
-        console.log("Updating", videoHeight);
-      }
-    };
+      window.addEventListener("resize", updateHeight);
 
-    window.addEventListener("resize", updateHeight);
-
-    return () => {
-      window.removeEventListener("resize", updateHeight);
-    };
+      return () => {
+        window.removeEventListener("resize", updateHeight);
+      };
+    }
   }, []);
 
   const getItemStyle = (sectionIndex: number, itemIndex: number) => {
```

**Review by bvaughn — APPROVED** [review]

File: src/ui/components/Passport/Passport.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/Passport/Passport.tsx:306]: Thank you! Yeah, I changed so many things I lost the plot.

**Comment by jonbell-lot23** [comment]
I cleaned up the duplicate code based on Brian's suggestion, and it seems fine for me on localhost. Committing and looking for a new review.

**Comment by jonbell-lot23** [comment]
Will file a ticket to move fully to css modules

**Pull request merged**